sdk/python/pvsite_datamodel/read/status.py

```diff
@@ -0,0 +1,26 @@
+"""
+Functions to read from the Status table
+"""
+
+from typing import Optional
+
+from pvsite_datamodel.sqlmodels import StatusSQL
+from sqlalchemy.orm import Session
+
+
+def get_latest_status(session: Session) -> Optional[StatusSQL]:
+    """
+    Gets the latest entry in the status table. Returns None if there are no entries
+
+    :param session: database session
+    return: Latest StatusSQL object, or None
+    """
+
+    # Query status table, ordered by created time
+    query = session.query(StatusSQL)
+    query = query.order_by(StatusSQL.created_utc.desc())
+
+    # get the top result
+    latest_status = query.first()
+
```

Comment:
could have an option, to say if its empty, add one and return that.

Reply:
Why would be the use case for that? Sems like it has the potaential to be misleading to me. Also Adding things shuold be scoped to the `write` package

Reply:
yea, lets leave it for the moment - you are write